hanadb_exporter/main.py

```diff
@@ -18,7 +18,7 @@
 from prometheus_client.core import REGISTRY
 from prometheus_client import start_http_server
 
-from hanadb_exporter import exporter_factory
+import exporter_factory
```

Comment:
We should still continue using the absolute path here. Did you have problems with the original code?

Reply:
Yes I had, but I'm not sure why...